invenio_communities/templates/semantic-ui/invenio_communities/details/members/invitation_details.html

```diff
@@ -0,0 +1,44 @@
+{# -*- coding: utf-8 -*-
+
+  This file is part of Invenio.
+  Copyright (C) 2022 CERN.
+
+  Invenio is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+{% extends "invenio_requests/details/index.html" %}
+
+{% set title = _("Invitation details") %}
+
+{% set active_menu_item = 'invitations' %}
+
+{% block page_body %}
+  {{ super.super() }}
```

Comment:
I would leave a comment on why we do this

Reply:
I see your comment, but it describes what it does instead of why :) could you describe why we call the grandparent template instead ?